README.md

```diff
@@ -113,7 +113,7 @@ Current release: `1.0.0-beta`
 
 ### v1.0 TODO:
 
-- [ ] Update docs for Button component - **[$20](https://opencollective.com/react-native-elements#budget)**
+- [x] Update docs for Button component - **[$20](https://opencollective.com/react-native-elements#budget)** - @deadcoder0904
```

Comment:
let's get rid of the $20 too. Just don't forget to submit a expense on Open Collective